README.md

```diff
@@ -1,5 +1,5 @@
 # Regression Games Unity Bots
-[![Static Badge](https://img.shields.io/badge/Latest%20Version-0.0.26-blue)](https://docs.regression.gg)
+[![Static Badge](https://img.shields.io/badge/Latest%20Version-0.0.27-blue)](https://docs.regression.gg)
 [![Join our Discord](https://img.shields.io/badge/Join%20our%20Discord-8A2BE2)](https://discord.com/invite/925SYVse2H)
 [![Changelog](https://img.shields.io/badge/Visit%20the%20Changelog-orange)](https://docs.regression.gg/changelog)
 
```

Comment:
I feel like this file should have a release version history / change history in it.  I know we have one in our overall release in one of the other projects, but for Unity users I would expect to see that in this file or another .md file in the same folder like `version.md` or something like that.

Reply:
Good suggestion! Created a ticket for this and put it into a few cycles in the future: https://linear.app/regression-games/issue/REG-1974/maintain-a-copy-of-the-changelog-within-rgunitybots